addon/index.js

```diff
@@ -61,6 +61,16 @@ export class QueryParamAwarePretender extends Pretender {
     OPTIONS: new Map(),
   };
 
+  normalizeURLs = true;
```

Comment:
Can we add a quick example to https://github.com/rwjblue/pretender-query-param-handler/blob/main/README.md ?

Reply:
Done